core/examples/driver/main.cpp

```diff
@@ -288,7 +240,7 @@ int main(int /*argc*/, char** /*argv*/)
       {
         std::cout << std::right << std::setw(2) << (*moduleIter)->GetModuleId() << std::left << " | ";
         std::cout << std::setw(20) << (*moduleIter)->GetName() << " | ";
-        std::cout << std::setw(9) << ((*moduleIter)->IsLoaded() ? "LOADED" : "UNLOADED");
+        std::cout << std::setw(9) << ((*moduleIter)->IsLoaded() ? "ACTIVE" : "RESOLVED");
```

Comment:
It depends on what's trying to be accomplished by the status information displayed by this code.

If we want to show users bundle state with a fully supported lifecycle, "resolved" and "active" would be acceptable. Otherwise, "resolved" would be "stopped" and "active" would be "started" given our current level of lifecycle support.

Ultimately, IsLoaded() would probably be replaced by something like GetState(), which is a standard OSGi defined function, when there is a fully supported lifecycle.